packages/components/src/util/URLResolver.ts

```diff
@@ -327,6 +328,17 @@ export class URLResolver {
                 }
             }),
 
+            new ActionMapper('query', 'executeQuery', row => {
+                const url = row.get('url');
+                if (url) {
+                    const params = ActionURL.getParameters(url);
+                    if (params.schemaName && params.schemaName.toLowerCase() == 'inventory'
+                        && params.queryName && params.queryName.toLowerCase() == 'item') {
+                        return createProductUrl("freezerManager", undefined, AppURL.create("rd", "sampleItem", params['query.MaterialId~eq']))
```

Comment:
Use `FREEZER_MANAGER_PRODUCT_ID` as first param?

Reply:
Ah yes, that's in here now.